docs/feature-store/transformations.md

```diff
@@ -241,6 +241,63 @@ config = fstore.RunConfig(local=False, function=my_func, handler="ingest_handler
 fstore.ingest(feature_set, source, run_config=config, spark_context=spark_service_name)
 ```
 
+### Spark operator ingestion example
+When using spark operator execution the MLRun run execution details would be returned, allowing tracking of its status and results.
+
+The following code should be executed only once to build the spark job image before running the first ingest
+It may take a few minutes to prepare the image
+```python
+from mlrun.runtimes import Spark3Runtime
+Spark3Runtime.deploy_default_image()
+```
+
+Spark operator ingestion:
+```python
+# mlrun: start-code
+```
+```python
+from mlrun.feature_store.api import ingest
+def ingest_handler(context):
+    ingest(mlrun_context=context) # The handler function must call ingest with the mlrun_context
+```
+You can run your PySpark code for ingesting data into the feature store by adding:
```

Comment:
I'm not sure it makes sense for this sentence to be in the middle of the `# mlrun: start-code / end-code` block...